react/src/components/Boxes.module.css

```diff
@@ -0,0 +1,374 @@
+#mainBox {
+    /* UI Properties*/
+    background-color: #f1f1f1;
+}
+
+#container {
+    /* Content Layout Properties */
+    display: flex;
+    align-items: center;
+}
+
+#centerBoxContainer {
+    /* Self Layout Properties */
+    position: relative;
+}
+
+#centerBoxOutline {
+    /* Self Layout Properties */
+    width: 100%;
+    height: 100%;
+}
+
+#centerBox {
+    /* Self Layout Properties */
+    position: absolute;
+    top: 1px;
+    left: 1px;      
+
+    /* UI Properties */
+    background-color: #FFF;
+}
+
+#centerBoxInner p {
+    margin: 0;
+    text-align: left;
+    letter-spacing: 0px;
+}
+
+#centerBoxHeader {
+    /* Self Layout Properties */
+    width: 229px;
+    height: 29px;
+
+    /* UI Properties */
+    font-size: 24px;
+    font-family: 'Lato', sans-serif;
+    color: #428E9E;
+}
+
+#centerBoxText {
+    /* Self Layout Properties */
+    width: 276px;
+    height: 29px;
+
+    /* UI Properties */
+    color: #333333;
+}
+
+@media only screen and (max-width: 600px) {
+    
+    #mainBox {
+        /* Self Layout Properties */
+        margin: 0 auto;
+        height: 480px;
+        width: 100vw;
```

Comment:
Zastanowiłbym się czy nie da się tego zrobić inaczej niż na `100vw`. `100vw` to szerokość ekranu, w związku z czym nie bierze pod uwagę scrollbara albo tego, czy przeglądarka jest wlączona na cały ekran, albo czy ktoś nie ma włączonej konsoli z prawej strony.